model/error/payload.go

```diff
@@ -35,79 +32,18 @@ var (
 	processorEntry    = common.MapStr{"name": processorName, "event": errorDocType}
 )
 
-type Payload struct {
-	Service m.Service
-	System  *m.System
-	Process *m.Process
-	User    *m.User
-	Events  []*Event
-}
-
-func DecodePayload(raw map[string]interface{}) (m.Payload, error) {
+func DecodePayload(raw map[string]interface{}) ([]transform.Eventable, error) {
```

Comment:
OK. I was thinking that the `Payload` in `DecodePayload` refers to the fact that it takes a raw intake v1 _payload_ and decodes, but I can see how it could also mean that it returns an object that fulfills the `Payload` interface. I'll see what it looks like as `DecodeEvents`